cmd/tools/generate/generate.go

```diff
@@ -157,10 +169,34 @@ func generateDocker(path string, kind int) {
 	if err != nil {
 		panic(err)
 	}
+
 	if kind == harvest {
-		_, _ = fmt.Fprintf(os.Stderr,
-			"Start containers with:\n"+
-				color.Colorize("docker-compose -f "+opts.outputPath+" up -d --remove-orphans\n", color.Green))
+		if opts.admin {
+			httpsd := conf.Config.Admin.Httpsd.Listen
+
+			if httpsd == "" {
+				panic("Missing admin configuration in harvest configuration")
+			}
+			adminPort := 8887
+			if strings.Contains(httpsd, ":") {
+				h := strings.Split(httpsd, ":")
+				adminPort, err = strconv.Atoi(h[len(h)-1])
+				if err != nil {
+					panic(err)
```

Comment:
seems like the message logged could be the same as line 188?

Reply:
done.